flint/entities/solars.py

```diff
@@ -89,7 +89,9 @@ def infocard(self, markup='html') -> str:
 
         specifications = lookup(self.ids_info)
         try:
-            synopsis = lookup(self.ids_info + 1)
+            synopsis = lookup(interface.get_infocardmap().get(self.ids_info))
+            if not synopsis:
```

Comment:
Why is this check needed? Does the infocard map contain invalid ids or does it point to blank resources?

Reply:
If a given solar's `ids_info` is not present in `infocardmap.ini`, `synopsis` will be an empty string, so fall back to `ids_info` + 1. 

But now that I think about it, it's not very likely that a solar's infocard even has a synopsis if there is no entry in `infocardmap.ini`, so this might be redundant

Reply:
In that case, you could write it as `interface.get_infocard_map().get(self.ids_info, self.ids_info + 1)`

I agree the tradeoffs are unclear. `self.ids_info + 1` might be more likely to result in something random getting appended to the infocard than something useful.